docs/spark-queries.md

```diff
@@ -143,6 +145,22 @@ spark.read
     .load("path/to/table")
 ```
 
+```scala
+// time travel to tag historical-snapshot
+spark.read
+    .option("tag", "historical-snapshot")
+    .format("iceberg")
+    .load("path/to/table")
+```
+
+```scala
+// time travel to the head snapshot of audit-branch
+spark.read
+    .option("branch", "audit-branch")
```

Comment:
Can we specify the static variable to use? `SparkReadOptions.BRANCH`

Reply:
Sure, updated!